lib/build-time-components/fa-icon.js

```diff
@@ -33,7 +32,10 @@ module.exports = class FaIconComponent extends BuildTimeComponent {
       'dataIcon:data-icon',
       'dataPrefix:data-prefix',
     ];
-    this.layout`<path d={{iconPath}} fill=currentColor></path>`;
+    this.layout`
+      <title>{{title}}</title>
```

Comment:
For the build time icons, are we going to get `<title></title>` if no title is present?

Reply:
Yes. I tried everything I could think of to work around this. I decided it's better to have a blank title when you don't expect it than a missing title when you do.  I'm also reaching out to some folks for help. I can take this out of this PR though if you want.

Reply:
Oh no, leave it in. I agree with you. Is this full-on Handlebars?

Does this work?

```
{{if title}}
  <title>{{title}}</title>
{{/if}}
```

Reply:
Actually I think it's `{{#if}}`...been awhile since I've used it.

Reply:
I thought that would work and based on the docs at https://github.com/cibernox/ember-ast-helpers#layout it should have.  Alas, it doesn't.